packages/keystone/fetch.js

```diff
@@ -53,7 +59,20 @@ async function fetchWithLogger (url, options, extraAttrs) {
     const parentTaskId = executionContext.taskId
     const parentExecId = executionContext.execId
 
-    const { skipTracingHeaders } = extraAttrs
+    if (!options.headers) {
+        options.headers = {}
+    }
+
+    const originalHeaders = pickBy(options.headers)
+
+    const { skipTracingHeaders, skipXTargetHeader } = extraAttrs
+
+    if (!skipXTargetHeader && !options.headers['X-Target']) {
```

Comment:
Why do you need skip? I cannot imagine case, when you don't follow the convention

Reply:
When you are sending requests to dadata or other extrenal services